operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ControllerConfigurationOverrider.java

```diff
@@ -1,18 +1,19 @@
 package io.javaoperatorsdk.operator.api.config;
 
+import io.fabric8.kubernetes.client.CustomResource;
+import io.javaoperatorsdk.operator.processing.event.internal.CustomResourceEventFilter;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
-import io.fabric8.kubernetes.client.CustomResource;
-
 public class ControllerConfigurationOverrider<R extends CustomResource<?, ?>> {
 
   private String finalizer;
   private boolean generationAware;
   private final Set<String> namespaces;
   private RetryConfiguration retry;
   private String labelSelector;
+  private CustomResourceEventFilter<R> customResourcePredicate;
```

Comment:
Also a list of predicates. Also the name of the field should correlated with the type IMHO.

Reply:
Yes, the name here is a leftover from the previous iteration, will fix it